requirements.txt

```diff
@@ -5,3 +5,5 @@ dask
 click
 boto3
 locket
+six
+tblib
```

Comment:
Typically, when a new version is released to pypi, I will run `conda skeleton` and diff that against the conda recipe in the repo to find dependencies that have been added/removed since the last release. So, you can add/remove deps in between releases without worry.